g/gnome-remote-desktop/stone.yaml

```diff
@@ -0,0 +1,57 @@
+#
+# SPDX-FileCopyrightText: © 2020-2024 Serpent OS Developers
+#
+# SPDX-License-Identifier: MPL-2.0
+#
+name        : gnome-remote-desktop
+version     : '46.4'
+release     : 1
+homepage    : https://gitlab.gnome.org/GNOME/gnome-remote-desktop
+upstreams   :
+    - https://gitlab.gnome.org/GNOME/gnome-remote-desktop/-/archive/46.4/gnome-remote-desktop-46.4.tar.gz : 951435badc993eff6a5260f52653860427a9a7ab2a23ea37dc996de46dea2d9a
+summary     : Remote desktop daemon for GNOME using pipewire
+description : |
+    Remote desktop server which allows you to connect to your machine remotely
+license     :
+    - GPL-2.0-or-later
+builddeps   :
+    - binary(mutter)
+    - pkgconfig(cairo)
+    - pkgconfig(dbus-1)
+    - pkgconfig(epoxy)
+    - pkgconfig(fdk-aac)
+    - pkgconfig(ffnvcodec)
+    - pkgconfig(freerdp3)
+    - pkgconfig(freerdp-client3)
+    - pkgconfig(freerdp-server3)
```

Comment:
These are all provided by the same devel package, you don't need all of them.

Reply:
For sure, these are distinct dependencies looked up from the Meson side. I figured it was best practice to list them separately on the off chance they were provided by different packages in the future (most likely never a chance here)